### ISSUE TITLE
enum34 gets installed on Python 3.7

### ISSUE THREAD

**Issue by goodspark** [splitio/python-client#166]
```
python --version
3.7.4
pip install splitio_client==8.1.6
...
pip list
Package        Version   
-------------- ----------
certifi        2019.11.28
chardet        3.0.4     
docopt         0.6.2     
enum34         1.1.6     
future         0.18.2    
futures        3.1.1     
idna           2.8       
pip            19.3.1    
PyYAML         5.3       
requests       2.22.0    
setuptools     45.0.0    
six            1.14.0    
splitio-client 8.1.6     
urllib3        1.25.7    
wheel          0.33.6    
```

This causes other issues when I run my application in Python 3.7.

**Comment by mredolatti** [comment]
Hi Sam, thanks for reaching out. I'll take a look at the dependencies and
get back to you.
Martin

**Comment by goodspark** [comment]
Ping? Please check out the PR I opened.

**Comment by mredolatti** [comment]
Hi @goodspark, sorry for the delay on this one. I've just merged it into a branch of ourselves and sent it to our QA env. If everything goes well, we'll make a relase before end of week. Thank you very much!

**Comment by goodspark** [comment]
Thanks!

**Comment by mredolatti** [comment]
Hi! Your PR has been merged & version 8.1.7 is out. Please let me know if everything is working correctly for you.

Thank you very much for reaching out and providing a fix :)

**Comment by goodspark** [comment]
We're good to go here!

Python 3.7
```
(.venv) sam@sam-Q325UAR ~/t/splitpy3test> pip list
Package        Version   
-------------- ----------
certifi        2019.11.28
chardet        3.0.4     
docopt         0.6.2     
future         0.18.2    
idna           2.8       
pip            18.1      
PyYAML         5.3       
requests       2.22.0    
setuptools     45.1.0    
splitio-client 8.1.7     
urllib3        1.25.8    
wheel          0.33.6    
```

Python 2.7
```
(.venv) sam@sam-Q325UAR ~/t/splitpy2test> pip list
Package        Version   
-------------- ----------
certifi        2019.11.28
chardet        3.0.4     
docopt         0.6.2     
enum34         1.1.6     
future         0.18.2    
futures        3.3.0     
idna           2.8       
pip            18.1      
PyYAML         5.3       
requests       2.22.0    
setuptools     44.0.0    
six            1.14.0    
splitio-client 8.1.7     
urllib3        1.25.8    
wheel          0.33.6    
```

Thanks
